sass/base/_base.scss

```diff
@@ -315,11 +320,44 @@ div.about__sample1 {
       #e8331365,
       #ae5242cd
     ),
-    url("./img/conference2.jpg");
+    url("../img/conference2.jpg");
   background-size: cover;
   background-position: center;
 }
 
+.summit {
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+.take-look {
+  padding: 0 12%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.summit__title {
+  color: #272a31;
```

Comment:
The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design 🔥 Well done!